packages/cli/src/commands/create/release.ts

```diff
@@ -83,6 +85,17 @@ export const createReleaseCommand = async ({
 
   const { release, app, publisher } = await getConfigWithChecks(buildToolsPath);
 
+  const apkEntry = release.files.find((asset: any) => asset.purpose === "install")!;
+  const size = (await fs.promises.stat(apkEntry.uri)).size;
+  if (size > 100 * 1024 * 1024) {
+    showMessage(
+      "Apk too large!!",
+      "Since the app size is over 100 MBs, the app would by default install only on a metered connection\n" +
+      "dApp store currently only supports `arm64-v8a`. Its possible to submit an `arm64-v8a` variant instead of a combined apk which works on all architectures",
```

Comment:
I think we need to give more suggested actions than this. Perhaps point them to this page? https://developer.android.com/studio/build/shrink-code

Reply:
Yeah this feels too heavy handed, especially since many games will go above the 100mb mark. To me this really feels like something we should control at the review/portal level not the tooling level. 

Trying to thread the needle between recommending a certain apk size that is relevant to all possible cases is going to be difficult.

The only thing that I can conceive of possibly being beneficial is just a general "hey this is a large APK and may require wifi download" that is a warning message:

Title: APK over 100MB
Message: Your APK file is over 100mb, which may result in requiring WiFi or other metered download operations for end users.

I think it would be fine to remove this entirely, but the above message is the only thing I think could be included.